api/src/pcapi/core/offerers/models.py

```diff
@@ -721,16 +721,19 @@ def target(self) -> Target | None:
         return self.registration.target if self.registration else None
 
     @property
-    def opening_days(self) -> list[dict]:
-        opening_days = []
+    def opening_days(self) -> dict | None:
+        if not self.openingHours:
+            return None
 
+        opening_days = {}
```

Comment:
(détail) Tu peux le faire sur une ligne:
```suggestion
opening_days = {} if self.openingHours else None
```

Reply:
C'était pour être sûr que ça ne casse pas sur `for opening_hours in self.openingHours:` si jamais self.openingHours est `None`